modal_proto/api.proto

```diff
@@ -1350,13 +1361,13 @@ message FunctionData {
   }
   repeated RankedFunction ranked_functions = 18;
 
-  // When the function is a "grouped" one, this records the # of tasks we want
-  // to schedule in tandem.
-  uint32 _experimental_group_size = 19;
-
   Schedule schedule = 20;
 
   reserved 21;
+
+  bool untrusted = 27; // If set, the function will be run in an untrusted environment.
```

Comment:
Did we ever end up using this btw?

Reply:
Not yet, never trust @pawalt with anything